workers/unity/Packages/com.improbable.gdk.tools/EditorConfig.cs

```diff
@@ -0,0 +1,9 @@
+namespace Improbable.Gdk.Tools
+{
+    internal static class EditorConfig
```

Comment:
`Tools.EditorConfig` reads a bit strange, I recommend something that mentions editor / spatialOS menu priorities instead.